chapters/VK_EXT_debug_utils.txt

```diff
@@ -44,6 +44,14 @@ include::../api/protos/vkSetDebugUtilsObjectNameEXT.txt[]
 
 include::../validity/protos/vkSetDebugUtilsObjectNameEXT.txt[]
 
+.Valid Usage
+****
+  * pname:pNameInfo\->objectType must: not be ename:VK_OBJECT_TYPE_UNKNOWN
```

Comment:
If I change it, won't it break as soon as that change goes in?  Instead, if I leave it like this, it will just start working since that PR is likely going to go in soon, right?

Reply:
That should be so. The PR is ancient though. After merge need to check for visible `pname`s anyway...

Reply:
Okay, I'll switch it, then.